templates/react-native/components/foo/Form.js

```diff
@@ -1,48 +1,45 @@
 import React, { Component } from 'react';
 import { Field, reduxForm } from 'redux-form';
 import { View } from 'react-native';
-import {
-  FormLabel,
-  FormInput,
-  FormValidationMessage,
-  Button,
-} from 'react-native-elements';
+import { Input, Button } from 'react-native-elements';
 
 
 class Form extends Component {
 
   renderField(data) {
-    const hasError = data.meta.touched && !!data.meta.error;
+    
     data.input.value = data.input.value.toString();
+    
+    let keyboard = data.type === "number" ? keyboard = "numeric" : keyboard = "default"
+    
     return (
       <View>
-        <FormLabel labelStyle={ {color: 'gray', fontSize: 20} }>
-            {data.input.name}
-        </FormLabel>
-        <FormInput containerStyle={ {flexDirection:'row'} }
-                   inputStyle={ {color: 'black', flex:1} }
-                   {...data.input}
-                   step={data.step}
-                   required={data.required}
-                   placeholder={data.placeholder}
-                   id={`{{{lc}}}_${data.input.name}`}
-                   multiline={true}
-                   keyboardType='numeric'
+        <Input
+      labelStyle={ {color: 'gray', flex:1} }
+      {...data.input}
+      required={data.required}
+      placeholder={data.placeholder}
+      id={`{{{lc}}}_${data.input.name}`}
+      label={data.input.name}
+      multiline={true}
+      errorMessage={data.meta.error}
+      keyboardType={keyboard}
         />
-        {hasError &&
-          <FormValidationMessage>{data.meta.error}</FormValidationMessage> }
+
       </View>
     );
   }
 
+  intParser = (value) => value ? parseInt(value, 10) : '';
```

Comment:
not sure about this, should this be used on every form?

Reply:
For me that was the only way to handle numbers. I think i need more investigations since i used react native elements 1.x it drop some issue in the template in general with actions.pop() etc.
So maybe we can get rid of that for the time beeing.

Reply:
I think that's better this way : 
`intParser = (value) => isNaN(value) ? value : parseInt(value, 10);`

Reply:
I know that's a bit messy but it should do the trick for now :

```
>  intParser = (value) => {
>     if(isNaN(value)) {
>       value
>     }
>     else if(this.props.initialValues) {
>       if ((isNaN(value)) && (typeof value === 'string')){
>          value
>       } else if(value) {
>         return parseInt(value, 10)
>       } else{
>         value
>       }
>     } 
>     else {
>       return parseInt(value, 10)
>     } 
>     return value
>   }
```

work in update & create